gkeepapi/__init__.py

```diff
@@ -401,7 +408,26 @@ def login(self, username, password, state=None, sync=True):
         """
         auth = APIAuth(self.OAUTH_SCOPES)
 
-        ret = auth.login(username, password, get_mac())
+        ret = auth.login(username=username, password=password, android_id=get_mac())
+        if ret:
+            self.load(auth, state, sync)
+
+        return ret
+
+    def login_with_master_token(self, email, master_token, state=None, sync=True):
+        """Authenticate to Google with the provided master token & sync.
+
+        Args:
+            email (str): The account to use.
+            master_token (str): The Master Token.
+            state (dict): Serialized state to load.
+
+        Raises:
+            LoginException: If there was a problem logging in.
+        """
+        auth = APIAuth(self.OAUTH_SCOPES)
+
+        ret = auth.login(master_token=master_token, android_id=get_mac())
```

Comment:
Could we create `set*` methods for each field (`android_id`, `email`) or move them to the constructor. Possibly a `load` method on `APIAuth` that also accepts the master token & sets up?